src/dso_api/dynamic_api/app_config.py

```diff
@@ -47,14 +47,24 @@ def register_model(self, model):
 
 def add_custom_serializers():
     from rest_framework.serializers import ModelSerializer
-    from schematools.contrib.django.models import LooseRelationField
-    from dso_api.dynamic_api.fields import LooseRelationUrlField
+    from schematools.contrib.django.models import (
+        LooseRelationField,
+        LooseRelationManyToManyField,
+    )
+    from dso_api.dynamic_api.fields import (
+        LooseRelationUrlField,
+        LooseRelationUrlListField,
+    )
 
     field_mapping = ModelSerializer.serializer_field_mapping
     field_mapping.update(
         {LooseRelationField: LooseRelationUrlField},
     )
 
+    field_mapping.update(
+        {LooseRelationManyToManyField: LooseRelationUrlListField},
+    )
+
```

Comment:
Dit update dus de globale `ModelSerializer.serializer_field_mapping` omdat je geen copie maakt!!
zou je daarnaast niet beide update statements combineren? Nu is het een inefficiente schrijfwijze van `dict[key] = value`

Reply:
Dat was inderdaad de bedoeling. Ik herschrijf hem naar een enkele update.

Reply:
Momenteel bevat de serializer_field_mapping alle mogeljke field mappings, ook degene die voor een specifieke dynamisch gegenereerde serializer misschien niet nodig zijn.